coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/CoralTrinoConfigKeys.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2021 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino;
+
+/**
+ * Contains configuration keys for {@link RelToTrinoConverter#configs}
+ */
+public class CoralTrinoConfigKeys {
+  public static final String SUPPORT_LEGACY_UNNEST_ARRAY_OF_STRUCT = "SUPPORT_LEGACY_UNNEST_ARRAY_OF_STRUCT";
```

Comment:
Please add documentation on what each of these config key means since this is a central place where those keys are stored.